backend/src/test/java/fk/prof/backend/test/ModelSerializationTest.java

```diff
@@ -1,173 +1,173 @@
-        assertEquals(10, wsch.getVmsLive());
-        assertEquals(1, wsch.getProfilesDiscarded());
-        assertEquals(3, wsch.getFailuresObserved());
-        assertEquals(10.0f, wsch.getEffectiveVmidCoveragePct(), 0.000001f);
-        assertEquals(exWsch.getLastScheduled(), wsch.getLastScheduled());
-
-        assertEquals(exWsch.getScheduling().getAfter(), wsch.getScheduling().getAfter());
-        assertEquals(60, wsch.getScheduling().getDuration());
-        assertEquals(300, wsch.getScheduling().getInterval());
-        assertEquals(25.0f, wsch.getScheduling().getVmidCoveragePct(), 0.000001f);
-    }
-
-    private WorkSchedule buildWorkSchedule() {
-        LocalDateTime someTime = LocalDateTime.of(2017, 1, 10, 10, 10, 10);
-        WorkSchedule.Schedule schedule = new WorkSchedule.Schedule(60 ,300, someTime, 25.0f);
-        List<WorkDetails> work = new ArrayList<>();
-        work.add(new WorkDetails(WorkType.cpu_sample, new WorkDetails.CpuSamplingAttributes(100, 10)));
-        work.add(new WorkDetails(WorkType.monitor_contention, new WorkDetails.MonitorContentionAttributes(100, 10)));
-
-        WorkSchedule wsch = new WorkSchedule(schedule, work, 5, 10, 1, 3, 10.0f, someTime.plusMinutes(10));
-
-        return wsch;
-    }
-
-    private void assertKeyValue(Map<String, Object> map, String key, Object value) {
-        assertTrue(map.containsKey(key));
-        if(value instanceof Float) {
-            final float epsilon = 0.0000001f;
-            Double d = (Double) map.get(key);
-            assertEquals(d.floatValue(), ((Float) value).floatValue(), epsilon);
-        } else {
-            assertEquals(value, map.get(key));
-        }
-    }
-
-    final String workScheduleSerialized = "{\n" +
-            "  \"scheduling\": {\n" +
-            "    \"duration\": 60,\n" +
-            "    \"interval\": 300,\n" +
-            "    \"vmid_coverage_pct\": 25.0,\n" +
-            "    \"after\": \"2017-01-10T10:10:10\"\n" +
-            "  },\n" +
-            "  \"work\": [\n" +
-            "    {\n" +
-            "      \"type\": \"cpu_sample\",\n" +
-            "      \"attributes\": {\n" +
-            "        \"frequency\": 100,\n" +
-            "        \"max_stack_depth\": 10\n" +
-            "      }\n" +
-            "    },\n" +
-            "    {\n" +
-            "      \"type\": \"monitor_contention\",\n" +
-            "      \"attributes\": {\n" +
-            "        \"max_monitor_count\": 100,\n" +
-            "        \"max_stack_depth\": 10\n" +
-            "      }\n" +
-            "    }\n" +
-            "  ],\n" +
-            "  \"vms_coverage\": 5,\n" +
-            "  \"vms_live\": 10,\n" +
-            "  \"profiles_discarded\": 1,\n" +
-            "  \"failures_observed\": 3,\n" +
-            "  \"effective_vmid_coverage_pct\": 10.0,\n" +
-            "  \"last_scheduled\": \"2017-01-10T10:20:10\"\n" +
-            "}";
-}
+//package fk.prof.backend.test;
```

Comment:
There were some jackson dependency issues. I pushed a newer commit where this is fixed. outdated diff.